pinot-perf/pom.xml

```diff
@@ -214,7 +214,6 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-shade-plugin</artifactId>
-        <version>3.1.0</version>
```

Comment:
Do we need to keep this plugin? I guess we should never package pinot-perf?

Reply:
this is not packaged to pinot-distribution, it's trying to generate another main class for perf testing.